include/nbl/builtin/hlsl/member_test_macros.hlsl

```diff
@@ -67,5 +73,94 @@ NBL_GENERATE_MEMBER_TESTER(y)
 NBL_GENERATE_MEMBER_TESTER(z)
 NBL_GENERATE_MEMBER_TESTER(w)
```

Comment:
can you add a `has_member_NAME<S,T>` version that takes a type of the member ? That one will probably be more useful.